README.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+
+This repository contains information related to the
+[CA/Browser Forum's](https://www.cabforum.org)
+[Plenary Organization](https://cabforum.org/about-us/) which governs the operations of the CABF as a whole.
```

Comment:
I'm not aware of us ever using the term "Plenary Organization" - do you have some references?

I'm wondering if this might be better as:

```suggestion
This repository contains information related to the
[CA/Browser Forum](https://www.cabforum.org) and the charters of specific
Chartered Working Groups which have been approved by the Forum.
```

Reply:
I don't--it's something we've used on calls and things to differentiate the CABF umbrella call from the SCWG call, but it doesn't have any basis in the Bylaws. I agree it's better to stick with the Bylaws terms for now until we can review those (we might introduce that term at some point as we do some revisions to finish cleaning up from the WG split-up).